test_opengraph_standards.py

```diff
@@ -0,0 +1,13 @@
+import requests
+import pytest
+from bs4 import BeautifulSoup
+
+
+class TestOpenGraphStandards:
```

Comment:
This is still incorrect. There should be no class TestOpenGraphStandards here. That should be in your test file with your test method. Fixtures should stand alone in a conftest.py file. What happened to those changes we discussed?